sdk/cognitiveservices/Knowledge.QnAMaker/src/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj

```diff
@@ -3,7 +3,7 @@
     <AssemblyTitle>Microsoft Cognitive Services QnAMaker SDK</AssemblyTitle>
     <AssemblyDescription>Provides access to the Microsoft Cognitive Services QnAMaker APIs.</AssemblyDescription>
     <Description>This client library provides access to the Microsoft Cognitive Services QnAMaker APIs.</Description>
-    <Version>1.1.0-preview.1</Version>
+    <Version>2.0.0-preview.1</Version>
```

Comment:
Any reason to go from 1.1 to 2.0 instead of sat 1.2?

Reply:
Yes some of the Async methods are missing that why I bumped the major version